opm/simulators/linalg/bda/clSources.cpp

```diff
@@ -0,0 +1,977 @@
+// This file is auto-generated. Do not edit!
```

Comment:
Generated files should not be commited and be put into the build directory. Please remove